app/controllers/rents_controller.rb

```diff
@@ -10,11 +11,16 @@ def index
   end
 
   def create
-    book = Book.find(params[:rent][:book_id])
+    @book = Book.find(params[:rent][:book_id])
+    @user = current_user
     render json: Rent.create(user: current_user, book: book,
                              from: params[:rent][:from], to: params[:rent][:to])
   rescue ActiveRecord::RecordNotFound
     render json: { error: 'The book you specified was not found' },
            status: :unprocessable_entity
   end
+
+  def sendmail
```

Comment:
I think you should test that an async job was created here.

Reply:
Done